src/app/types/record-works.endpoint.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  ExternalIdentifier,
+  MonthDayYearDate,
+  Value,
+  Visibility,
+  VisibilityStrings,
+} from './common.endpoint'
+
+export interface Work {
```

Comment:
@leomendoza123 I think we should extend the AssertionBase and just put the unique Work attributes